src/measure_theory/arithmetic.lean

```diff
@@ -91,28 +103,31 @@ instance has_measurable_mul₂.to_has_measurable_mul [has_measurable_mul₂ M] :
   has_measurable_mul M :=
 ⟨λ c, measurable_const.mul measurable_id, λ c, measurable_id.mul measurable_const⟩
 
-@[to_additive]
+@[to_additive, measurability]
 lemma measurable.const_mul [has_measurable_mul M] {f : α → M} (hf : measurable f) (c : M) :
   measurable (λ x, c * f x) :=
 (measurable_const_mul c).comp hf
 
-@[to_additive]
+@[to_additive, measurability]
 lemma ae_measurable.const_mul [has_measurable_mul M] {f : α → M} {μ : measure α}
   (hf : ae_measurable f μ) (c : M) :
   ae_measurable (λ x, c * f x) μ :=
 (has_measurable_mul.measurable_const_mul c).comp_ae_measurable hf
 
-@[to_additive]
+@[to_additive, measurability]
 lemma measurable.mul_const [has_measurable_mul M] {f : α → M} (hf : measurable f) (c : M) :
   measurable (λ x, f x * c) :=
 (measurable_mul_const c).comp hf
 
-@[to_additive]
+@[to_additive, measurability]
 lemma ae_measurable.mul_const [has_measurable_mul M] {f : α → M} {μ : measure α}
   (hf : ae_measurable f μ) (c : M) :
   ae_measurable (λ x, f x * c) μ :=
 (measurable_mul_const c).comp_ae_measurable hf
 
+attribute [measurability] measurable.add' measurable.add ae_measurable.add ae_measurable.add'
```

Comment:
does switching the order on the above attrs to `@[measurability, to_additive]` work instead of this?

Reply:
I will try. The continuity tactic doc states "`to_additive` doesn't know yet how to copy the attribute to the additive version" and I trusted it blindly.

Reply:
ahh, yep it seems like it won't work: https://github.com/leanprover-community/mathlib/blob/17f3b801f8981c499b0c13c7e05e4588a9570b31/src/topology/algebra/monoid.lean#L51

Reply:
I tested it. It does not work.